### ISSUE TITLE
Receive tab expires option is a source of confusion

### ISSUE THREAD

**Issue by AbdussamadA** [spesmilo/electrum#2042]
Hi

So I keep seeing this on the forum and on reddit. New users think that the expires option on the receive tab means that they have to receive payment within a certain time and if they don't their money is lost. The most recent example:

>And under the receive tab, since my transaction is still pending, will I loose all my money in 23 hours when the request expires?

https://bitcointalk.org/index.php?topic=1683650.0

Why is there a request expiry option? What is its purpose and can it be removed? It is a source of endless confusion.

**Comment by RealBTC** [comment]
This is why the address tab should be shown by default as I mentioned in issue #2036.

That receive payment is stupid, bitcoin is a push transaction not a pull transaction, we should distinguish bitcoin from banking transactions because its much different.

**Comment by dabura667** [comment]
1. I agree the expiration date is unclear as to its meaning. Either make it clear or get rid of it. +1
2. I disagree that we should show the address tab as default. We should have a UI that encourages newbies to use a new address for every single time they want to receive bitcoin. The "invoice" feature is one way to do that, but it is slightly misleading (see 1)

tl;dr

Yes, we should get rid of expiry date on invoices.

**Comment by ecdsa** [comment]
the purpose is to protect the payee against volatility.

**Comment by ecdsa** [comment]
I will set the default to 'never', I guess it should calm down newbies concerns

**Comment by ecdsa** [comment]
commit f381aee83549a456a47f2d2a001f7215023a3117
